scripts/tests/web.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -eu # don't use -x as it will leak the JWT
+
+earthly=${earthly:=earthly}
+earthly=$(realpath "$earthly")
+echo "running tests with $earthly"
+
+## Test that we display web link correctly without being logged in and with no params
+NO_COLOR=0 "$earthly" web 2>&1 | grep -qE "https://ci(-beta)?.earthly.dev/login"
+RESULT=$?
+if [ $RESULT -ne 0 ]; then
+  echo "failed without login and no arguments"
+  exit $RESULT
+fi
+
+# Test with argument passed for provider
+NO_COLOR=0 "$earthly" web --provider github 2>output-file
+output=$(cat output-file)
+rm output-file
+
+RESULT=$?
+if [ $RESULT -ne 0 ]; then
+  echo "failed without login provider argument passed"
```

Comment:
in addition to there being a `set -e`, this appears to be testing if the `rm` failed.